doc/release_notes/release_notes_1.5.rst

```diff
@@ -0,0 +1,328 @@
+- :acrn-issue: '4074' -[KBL][acrn-configuration-tool]: Cx desc parsing enhancement
+- :acrn-issue: '4082' -[acrn-configuration-tool]bypass acpi_idle/acpi_cpufreq driver
+- :acrn-issue: '4094' -Error parameter for intel_pstate in launch_hard_rt_vm.sh
+- :acrn-issue: '4099' -[Community][External]Boot issue on non Apollo/Kaby lake.
+- :acrn-issue: '4116' -[Community][External]How to set CPU Core UOS
+- :acrn-issue: '4123' -[Community][External]Creating Ubuntu SOS not working - black screen
+- :acrn-issue: '4125' -[Community][External]vm1 is running, can't create twice!
+- :acrn-issue: '4128' -[WHL][acrn-configuration-tool]WebUI can not select vuart 0&vuart 1 by default
+- :acrn-issue: '4135' -[Community][External]Invalid guest vCPUs (0) Ubuntu as SOS.
+- :acrn-issue: '4139' -[Community][External]mngr_client_new: Failed to accept from fd 38
+- :acrn-issue: '4143' -[acrn-configuration-tool] bus of DRHD scope devices is parsed incorrectly
+- :acrn-issue: '4163' -[acrn-configuration-tool] not support: –s n,virtio-input
+- :acrn-issue: '4164' -[acrn-configuration-tool] not support: –s n,xhci,1-1:1-2:2-1:2-2
+- :acrn-issue: '4165' -[WHL][acrn-configuration-tool]Configure epc_section is incorrect
+- :acrn-issue: '4172' -[acrn-configuration-tool] not support: –s n,virtio-blk, (/root/part.img---dd if=/dev/zero of=/root/part.img bs=1M count=10  all/part of img, one u-disk device, u-disk as rootfs and the n is special)
+- :acrn-issue: '4173' -[acrn-configuartion-tool]acrn-config tool not support parse default pci mmcfg base
+- :acrn-issue: '4175' -acrntrace fixes and improvement
+- :acrn-issue: '4185' -[acrn-configuration-tool] not support: –s n,virtio-net, (not set,error net, set 1 net, set multi-net, vhost net)
+- :acrn-issue: '4211' -[kbl nuc] acrn failed to boot when generate hypervisor config source from config app with HT enabled in BIOS
+- :acrn-issue: '4212' -[KBL][acrn-configuration-tool][WaaG+RTVM]Need support pm_channel&pm_by_vuart setting for Board:nuc7i7dnb+WaaG&RTVM
+- :acrn-issue: '4227' -[ISD][Stability][WaaG][Regression] "Passmark8.0-Graphics3D-DirectX9Complex" test failed on WaaG due to driver error
+- :acrn-issue: '4228' -[acrn-configuration-tool] cannot boot hypervisor on customer board with KBL 7300U
+- :acrn-issue: '4229' -Add range check in Kconfig.
+- :acrn-issue: '4230' -Remove MAX_VCPUS_PER_VM in Kconfig
+- :acrn-issue: '4232' -Set default KATA_VM_NUM to 1 for SDC
+- :acrn-issue: '4247' -[acrn-configuration-tool] Generate Scenario for VM0 communites with VM1 is incorrect.
+- :acrn-issue: '4249' -[acrn-configuration-tool]Generated Launchscript but WebUI prompt error msg after we just select passthru-devices:audio_codec
+- :acrn-issue: '4255' -[acrn-configuration-tool][nuc7i7dnb][sdc]uos has no ip address
+- :acrn-issue: '4260' -[Community][External]webcam switch between 2 UOS.
+- :acrn-issue: '4286' -[acrn-configuration-tool] Remove VM1.vcpu_affinity.pcuid=3 for VM1 in sdc scenario
+
+Known Issues
+************
+TBD
```

Comment:
If no Known Issues exist, please change this to None (instead of TBD).

Reply:
Waiting for final clear linux version.